mmv1/third_party/terraform/tests/resource_healthcare_fhir_store_test.go.erb

```diff
@@ -211,3 +237,43 @@ func testAccCheckGoogleHealthcareFhirStoreUpdate(t *testing.T, pubsubTopic strin
 		return nil
 	}
 }
+
+func testAccHealthcareFhirStore_healthcareFhirStoreNotificationConfigExample(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_healthcare_fhir_store" "default" {
+  name    = "tf-test-example-fhir-store%{random_suffix}"
+  dataset = google_healthcare_dataset.dataset.id
+  version = "R4"
+
+  enable_update_create          = false
+  disable_referential_integrity = false
+  disable_resource_versioning   = false
+  enable_history_import         = false
+
+  labels = {
+    label1 = "labelvalue1"
+  }
+
+<% if version == "beta" -%>
```

Comment:
Rather than version guarding the field, let's version guard the test case as a whole! That way there are two separate tests: one for the GA field (run at both GA + beta), and one for the beta field (run only at beta)